pkg/pillar/types/zedroutertypes.go

```diff
@@ -313,21 +314,21 @@ type DeviceNetworkStatus struct {
 
 func (status *DeviceNetworkStatus) GetPortByName(
 	port string) *NetworkPortStatus {
-	for _, portStatus := range status.Ports {
+	for i, portStatus := range status.Ports {
 		if strings.EqualFold(portStatus.Name, port) {
 			log.Infof("Found NetworkPortStatus for %s", port)
```

Comment:
While u r here, can you remove this log.Info statement? Doesn't make sense here..